### PR TITLE
Add validation for system general IP fields

### PR DESCRIPTION
**Pull request by erinix**
NAS-103423

### CODE REVIEW

File: src/app/helptext/system/general.ts [hunk 1]
```diff
@@ -188,5 +188,10 @@ export const helptext_system_general = {
   snackbar_download_success: {
     title: T("Download Sucessful"),
     action: T("Success")
+  },
+
+  validation_errors: {
+    ui_address: T("When 0.0.0.0 has been selected, selection of other addresses is not allowed."),
```
Comment (wblock) [inline src/app/helptext/system/general.ts]: I don't know if this works as a general error.  Does selecting 0.0.0.0 or :: always mean "all addresses"?  If it does, we should say so: "When all has been selected by entering <samp>0.0.0.0</samp> or <samp>::</samp>, no other addresses can be entered."

Comment (erinix) [reply src/app/helptext/system/general.ts]: I believe so, you can either pick specific ip addresses or the 0.0.0.0 or :: wildcards

Comment (wblock) [reply src/app/helptext/system/general.ts]: Okay, how about:
"When all addresses have been selected by choosing <samp>0.0.0.0</samp> or <samp>::</samp>, additional addresses cannot be chosen."

File: src/app/helptext/system/general.ts [hunk 2]
```diff
@@ -188,5 +188,10 @@ export const helptext_system_general = {
   snackbar_download_success: {
     title: T("Download Sucessful"),
     action: T("Success")
+  },
+
+  validation_errors: {
+    ui_address: T("When all addresses have been selected by choosing 0.0.0.0, additional addresses cannot be chosen."),
```
Comment (wblock) [inline src/app/helptext/system/general.ts]:
```suggestion
    ui_address: T("Select <samp>0.0.0.0</samp> to include all addresses. When this has been chosen, additional addresses cannot be selected."),
```

**Pull request merged**